### PR TITLE
Clone nodes

### PR DESCRIPTION
**Pull request by OlyaRybak**

### CODE REVIEW

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/11/

**Build Log**
*last 25 lines*

 ```
[...truncated 7679 lines...]
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
changed: [localhost]

TASK: [Remove live search server] ********************************************* 
failed: [localhost] => {"changed": true, "cmd": "APP_ENV=dev drush eval 'search_api_server_delete('apache_solr');'", "delta": "0:00:04.295648", "end": "2015-10-02 10:15:02.926072", "rc": 255, "start": "2015-10-02 10:14:58.630424", "warnings": []}
stderr: PHP Fatal error:  Call to a member function delete() on boolean in /var/www/malmo_build11/profiles/mal/modules/contrib/search_api/search_api.module on line 2504
Drush command terminated abnormally due to an unrecoverable error.   [error]
Error: Call to a member function delete() on boolean in
/var/www/malmo_build11/profiles/mal/modules/contrib/search_api/search_api.module,
line 2504

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/home/jenkins/reinstall.retry

localhost                  : ok=22   changed=15   unreachable=0    failed=1   

Build step 'Execute shell' marked build as failure
```

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/12/

**Build Log**
*last 25 lines*

 ```
[...truncated 7677 lines...]

TASK: [Downloading registry_rebuild] ****************************************** 
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
failed: [localhost] => {"changed": true, "cmd": "drush rr", "delta": "0:00:01.698009", "end": "2015-10-02 11:23:36.883987", "rc": 255, "start": "2015-10-02 11:23:35.185978", "warnings": []}
stderr: The registry has been rebuilt via registry_rebuild (A).              [success]
PHP Parse error:  syntax error, unexpected ')' in /var/www/malmo_build12/profiles/mal/modules/custom/mal_update/mal_update.install on line 1021
Drush command terminated abnormally due to an unrecoverable error.   [error]
Error: syntax error, unexpected ')' in
/var/www/malmo_build12/profiles/mal/modules/custom/mal_update/mal_update.install,
line 1021

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/home/jenkins/reinstall.retry

localhost                  : ok=21   changed=14   unreachable=0    failed=1   

Build step 'Execute shell' marked build as failure
```

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>23</strong> JSHint: modules standard file http://128.199.38.86/malmo_build13/build_reports/modulesjshint.txt
<strong>3</strong> JSHint: themes standard file http://128.199.38.86/malmo_build13/build_reports/themesjshint.txt
<strong>6820</strong> JSHint: profile standard file http://128.199.38.86/malmo_build13/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build13
Behat tests: standard file http://128.199.38.86/malmo_build13/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/13/

**Comment by OlyaRybak** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>23</strong> JSHint: modules standard file http://128.199.38.86/malmo_build14/build_reports/modulesjshint.txt
<strong>3</strong> JSHint: themes standard file http://128.199.38.86/malmo_build14/build_reports/themesjshint.txt
<strong>6820</strong> JSHint: profile standard file http://128.199.38.86/malmo_build14/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build14
Behat tests: standard file http://128.199.38.86/malmo_build14/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/14/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/17/

**Build Log**
*last 25 lines*

 ```
[...truncated 7679 lines...]
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
changed: [localhost]

TASK: [Remove live search server] ********************************************* 
failed: [localhost] => {"changed": true, "cmd": "APP_ENV=dev drush eval 'search_api_server_delete('apache_solr');'", "delta": "0:00:04.708039", "end": "2015-10-05 04:18:07.306177", "rc": 255, "start": "2015-10-05 04:18:02.598138", "warnings": []}
stderr: PHP Fatal error:  Call to a member function delete() on boolean in /var/www/malmo_build17/profiles/mal/modules/contrib/search_api/search_api.module on line 2504
Drush command terminated abnormally due to an unrecoverable error.   [error]
Error: Call to a member function delete() on boolean in
/var/www/malmo_build17/profiles/mal/modules/contrib/search_api/search_api.module,
line 2504

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/home/jenkins/reinstall.retry

localhost                  : ok=22   changed=15   unreachable=0    failed=1   

Build step 'Execute shell' marked build as failure
```

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>23</strong> JSHint: modules standard file http://128.199.38.86/malmo_build20/build_reports/modulesjshint.txt
<strong>3</strong> JSHint: themes standard file http://128.199.38.86/malmo_build20/build_reports/themesjshint.txt
<strong>6820</strong> JSHint: profile standard file http://128.199.38.86/malmo_build20/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build20
Behat tests: standard file http://128.199.38.86/malmo_build20/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/20/

File: drupal/blk.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# You should install ansible for ability to run this script
+# sudo apt-get install software-properties-common
+# sudo apt-add-repository ppa:ansible/ansible
+# sudo apt-get update
+# sudo apt-get install ansible
+# sudo apt-get install python-mysqldb
+time ansible-playbook reinstall.yml -i 'localhost,' --connection=local --extra-vars "mysql_user=root mysql_pass=penis242
+mysql_db=olya_malmo mysql_host=mysql.loc
```
Comment (Mitsuroseba) [inline drupal/blk.sh]: delete this file from repo and add into gitignore ?

File: drupal/profiles/mal/modules/custom/mal_update/mal_update.install [hunk 2]
```diff
@@ -998,7 +998,25 @@ function mal_update_update_7078() {
             'content_type_course_packages',
             'content_type_course_template',
     // View own unpublished content.
-            'config_permissions',
+            'config_permissions',git
```
Comment (Mitsuroseba) [inline drupal/profiles/mal/modules/custom/mal_update/mal_update.install]: delete this typo

File: drupal/profiles/mal/modules/custom/mal_update/mal_update.install [hunk 3]
```diff
@@ -998,7 +998,25 @@ function mal_update_update_7078() {
...
   ) as $module)
   features_revert_module($module);
 }
+
+/**
+ * Clone nodes.
+ */
+function mal_update_update_7080() {
```
Comment (Mitsuroseba) [inline drupal/profiles/mal/modules/custom/mal_update/mal_update.install]: mal_update_update_7079 ?

**Comment by Mitsuroseba** [comment]
:-1:

File: drupal/profiles/mal/modules/custom/mal_update/mal_update.install (hunk 3 above)
Comment (OlyaRybak) [inline drupal/profiles/mal/modules/custom/mal_update/mal_update.install]: 7079 is used by Pasha in his commit

Comment (Mitsuroseba) [inline drupal/profiles/mal/modules/custom/mal_update/mal_update.install]: Got it! :)

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>23</strong> JSHint: modules standard file http://128.199.38.86/malmo_build21/build_reports/modulesjshint.txt
<strong>3</strong> JSHint: themes standard file http://128.199.38.86/malmo_build21/build_reports/themesjshint.txt
<strong>6820</strong> JSHint: profile standard file http://128.199.38.86/malmo_build21/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build21
Behat tests: standard file http://128.199.38.86/malmo_build21/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/21/

**Comment by Mitsuroseba** [comment]
:+1:

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>23</strong> JSHint: modules standard file http://128.199.38.86/malmo_build22/build_reports/modulesjshint.txt
<strong>3</strong> JSHint: themes standard file http://128.199.38.86/malmo_build22/build_reports/themesjshint.txt
<strong>6820</strong> JSHint: profile standard file http://128.199.38.86/malmo_build22/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build22
Behat tests: standard file http://128.199.38.86/malmo_build22/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/22/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/27/

**Build Log**
*last 25 lines*

 ```
[...truncated 7758 lines...]

TASK: [Install apt repos] ***************************************************** 
ok: [localhost] => (item=ppa:chris-lea/node.js)

TASK: [Update apt cache] ****************************************************** 
ok: [localhost]

TASK: [Install apt packages] ************************************************** 
failed: [localhost] => (item=ruby1.9.1,ruby1.9.1-dev,rubygems1.9.1,irb1.9.1,ri1.9.1,rdoc1.9.1,build-essential,libopenssl-ruby1.9.1,libssl-dev,zlib1g-dev,nodejs) => {"failed": true, "item": "ruby1.9.1,ruby1.9.1-dev,rubygems1.9.1,irb1.9.1,ri1.9.1,rdoc1.9.1,build-essential,libopenssl-ruby1.9.1,libssl-dev,zlib1g-dev,nodejs"}
stderr: E: Could not get lock /var/lib/dpkg/lock - open (11: Resource temporarily unavailable)
E: Unable to lock the administration directory (/var/lib/dpkg/), is another process using it?

msg: '/usr/bin/apt-get -y -o "Dpkg::Options::=--force-confdef" -o "Dpkg::Options::=--force-confold"   install 'rubygems1.9.1' 'irb1.9.1' 'rdoc1.9.1' 'libopenssl-ruby1.9.1'' failed: E: Could not get lock /var/lib/dpkg/lock - open (11: Resource temporarily unavailable)
E: Unable to lock the administration directory (/var/lib/dpkg/), is another process using it?


FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/home/jenkins/sniffers.retry

localhost                  : ok=3    changed=0    unreachable=0    failed=1   

Build step 'Execute shell' marked build as failure
```

**Comment by OlyaRybak** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>23</strong> JSHint: modules standard file http://128.199.38.86/malmo_build28/build_reports/modulesjshint.txt
<strong>3</strong> JSHint: themes standard file http://128.199.38.86/malmo_build28/build_reports/themesjshint.txt
<strong>6820</strong> JSHint: profile standard file http://128.199.38.86/malmo_build28/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build28
Behat tests: standard file http://128.199.38.86/malmo_build28/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/28/

**Pull request merged**